packages/ra-ui-materialui/src/input/TextInput.tsx

```diff
@@ -72,6 +73,8 @@ export const TextInput: FunctionComponent<
                     helperText={helperText}
                 />
             }
+            multiline={multiline}
```

Comment:
Please revert all changes in this file. We don't want to replicate all props of the underlying TextField compnent

Reply:
Done